src/map/homunculus.cpp

```diff
@@ -1138,6 +1121,8 @@ bool hom_call(map_session_data *sd)
 		clif_send_homdata( *hd, SP_ACK );
 		clif_hominfo(sd,hd,1);
 		clif_hominfo(sd,hd,0); // send this x2. dunno why, but kRO does that [blackhole89]
+		clif_homunculus_updatestatus(*sd, SP_HP);
+		clif_homunculus_updatestatus(*sd, SP_SP);
```

Comment:
Why do we need to resend here?
HP/SP are part of clif_hominfo.

Reply:
Same as other cases, official servers call them after the 2 hominfo